### PR TITLE
add familysearch.org/records/r2 to blueprint

### PR DESCRIPTION
**Pull request by brinbh**
This is a URL that will be used for the Records Improvement Indexing Lite (RII) which is a prototype allowing users to index any image found in Explore Images with the assistance of automated indexing.

### CODE REVIEW

File: blueprint.yml [hunk 1]
```diff
@@ -16,17 +16,63 @@ deploy:
       location: development-fh5-useast1-heroku
       binding_sets:
         records-improvement.frontier: # temporary path /frontier/records-improvement until a more permanent path is established
```
Comment (patrickliechty) [inline blueprint.yml:18]: Line 18 needs to be changed to r2.records to be consistent with the other refs.

Comment (brinbh) [reply blueprint.yml:18]: No, that is the original url that we had for testing the image viewer to react. That one can stay. I believe it is line 30 you were probably thinking about.

Comment (patrickliechty) [reply blueprint.yml:18]: Ok, why do you want to keep the old one?

Comment (patrickliechty) [reply blueprint.yml:18]: Why do you want to keep records-improvement.frontier?

Comment (brinbh) [reply blueprint.yml:18]: It is good for testing those components, but I guess we don't really need that anymore

**Pull request merged**